packages/framework/src/Resources/config/services.yml

```diff
@@ -985,3 +985,21 @@ services:
     Shopsys\FrameworkBundle\Model\Order\Mail\OrderMail: ~
 
     Shopsys\FrameworkBundle\Model\Product\Listing\RequestToOrderingModeIdConverter: ~
+
+    productAdvancedSearchQueryBuilderExtender:
+        class: Shopsys\FrameworkBundle\Model\AdvancedSearch\AdvancedSearchQueryBuilderExtender
+        arguments:
+            $advancedSearchConfig: '@Shopsys\FrameworkBundle\Model\AdvancedSearch\ProductAdvancedSearchConfig'
+
+    Shopsys\FrameworkBundle\Model\AdvancedSearch\AdvancedSearchFacade:
```

Comment:
I would love if you renamed the class to `AdvancedSearchProductFacade`, the naming drives me crazy :laughing:

Reply:
created issue for phase 7 because it is something like polishing for me...